BitwardenShared/UI/Platform/Application/AppProcessorTests.swift

```diff
@@ -200,6 +200,46 @@ class AppProcessorTests: BitwardenTestCase { // swiftlint:disable:this type_body
         XCTAssertIdentical(syncService.delegate, subject)
     }
 
+    /// `handleAppLinks(URL)` navigates the user based on the input URL.
+    func test_init_handleAppLinks() {
+        let url = URL(string:
+            "https://bitwarden.com/finish-signup?email=example@email.com&token=verificationtoken&fromEmail=true"
+        )
+        subject.handleAppLinks(incomingURL: url!)
+
+        XCTAssertEqual(coordinator.routes.last, .auth(.completeRegistrationFromAppLink(
+            emailVerificationToken: "verificationtoken",
+            userEmail: "example@email.com",
+            fromEmail: true,
+            region: .unitedStates
+        )))
+    }
+
+    /// `handleAppLinks(URL)` checks error report for `.appLinksInvalidURL`.
+    func test_init_handleAppLinks_invalidURL() {
```

Comment:
🎨 This is only testing one of the flows that can throw `invalidURL`, tests should cover the other flows as well.